src/assets/icon/IconWarningCircleBlue.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Svg, {Path, Rect} from 'react-native-svg';
+
+export default function IconWarningCircleBlue(props) {
+  return (
+    <Svg
+      width={24}
+      height={24}
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}>
+      <Path
+        opacity={1}
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M12 22c5.523 0 10-4.477 10-10S17.523 2 12 2 2 6.477 2 12s4.477 10 10 10z"
+        fill={props.fill || '#2F80ED'}
+      />
+      <Rect x={11} y={7} width={2} height={8} rx={1} fill={props.fillIcon || '#2F80ED'} />
+      <Rect x={11} y={16} width={2} height={2} rx={1} fill={props.fillIcon || '#2F80ED'} />
+    </Svg>
+  );
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `IconWarningCircleBlue` component seems to be well implemented. It uses SVG and allows for customization of fill color through props, which is a good practice for reusability. However, it's important to note that the default color `#2F80ED` is hardcoded in two places. If this color changes in the future, you'll have to remember to change it in both places. To avoid this, consider defining the default color as a constant at the top of your file or in a separate constants file.

```diff
+ const DEFAULT_FILL_COLOR = '#2F80ED';

  export default function IconWarningCircleBlue(props) {
    return (
      <Svg
        width={24}
        height={24}
        viewBox="0 0 24 24"
        fill="none"
        xmlns="http://www.w3.org/2000/svg"
        {...props}>
        <Path
          opacity={1}
          fillRule="evenodd"
          clipRule="evenodd"
          d="M12 22c5.523 0 10-4.477 10-10S17.523 2 12 2 2 6.477 2 12s4.477 10 10 10z"
-         fill={props.fill || '#2F80ED'}
+         fill={props.fill || DEFAULT_FILL_COLOR}
        />
        <Rect x={11} y={7} width={2} height={8} rx={1} 
-         fill={props.fillIcon || '#2F80ED'} 
+         fill={props.fillIcon || DEFAULT_FILL_COLOR}
        />
        <Rect x={11} y={16} width={2} height={2} rx={1} 
-         fill={props.fillIcon || '#2F80ED'} 
+         fill={props.fillIcon || DEFAULT_FILL_COLOR}
        />
      </Svg>
    );
  }
```
This way, if the default color needs to be changed, it only needs to be updated in one place. This makes the code more maintainable and less prone to errors.


<!-- This is an auto-generated comment by OSS CodeRabbit -->